pkg/reconciler/parallel/controller.go

```diff
@@ -48,12 +48,7 @@ func NewController(
 	subscriptionInformer := subscription.Get(ctx)
 	serviceaccountInformer := serviceaccountinformer.Get(ctx)
 
-	var globalResync func(obj interface{})
-	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"), func(name string, value interface{}) {
-		if globalResync != nil {
-			globalResync(nil)
-		}
-	})
```

Comment:
you still need the watcher for the features config map. But you need to set `globalResync`. E.g. after line 62, with something like the following:

```
globalResync = func(_ interface{}) {
	impl.GlobalResync(parallelInformer.Informer())
}
```

Reply:
ohhh yes, this is understandable. I must have declared the `globalResync` and mentioned the informer

Reply:
Yes. That way a reconcile of all parallels is triggered, when the feature configmaps changes (e.g. from oidc.auth disabled -> enabled, we must create the service accounts)